content/10.diagnostics.md

```diff
@@ -144,51 +144,51 @@ In some cases, library preparation can involve other modifications of the sample
 
 #### RT-LAMP
 
 RT-PCR remains the gold standard for detection of SARS-CoV-2 RNA from infected patients, but the traditional method requires special equipment and reagents, including a thermocycler.
-Loop-mediated isothermal amplification (LAMP) is an alternative to PCR that does not require specialized equipment like a thermocycler [@DOI:10.1093/nar/28.12.e63].
+Loop-mediated isothermal amplification (LAMP) is an alternative to PCR that does not require specialized equipment [@DOI:10.1093/nar/28.12.e63].
 In this method, nucleic acids are amplified in a 25 &mu;L reaction that is incubated and chilled on ice [@doi:10.1093/nar/28.12.e63].
 It uses primers designed to facilitate auto-cycling strand displacement DNA synthesis [@doi:10.1093/nar/28.12.e63].
 LAMP can be combined with reverse transcription (RT-LAMP) to enable the detection of RNA.
 
-One study showed that RT-LAMP is effective for detection of SARS-CoV-2 with excellent specificity and sensitivity, and that this method can be applied to unprocessed saliva samples [@doi:10.1038/s41598-021-95799-6].
+One study showed that RT-LAMP is effective for detection of SARS-CoV-2 with excellent specificity and sensitivity and that this method can be applied to unprocessed saliva samples [@doi:10.1038/s41598-021-95799-6].
 This method was benchmarked against RT-PCR using 177 human nasopharyngeal RNA samples, of which 126 were COVID positive.
 The authors break down the sensitivity of their test according to the C~t~ value from RT-PCR of the same samples; RT-LAMP performs at 100% sensitivity for samples with a C~t~ from RT-PCR of 32 or less.
 The performance is worse when considering all RT-PCR positive samples (including those with C~t~ values between 32-40).
 However, there is some evidence suggesting that samples obtained from individuals that achieve C~t~ values >30 measured using RT-PCR tend to be less infective that those that record a C~t~ value <30 [@doi:10.1016/j.jiph.2021.08.013; @doi:10.1017/ice.2021.132; @doi:10.1093/cid/ciaa619], so RT-LAMP is still a useful diagnostic tool.
-Various combinations of reagents are available, but one example of reagents being used is the WarmStart Colorimetric LAMP 2X Master Mix with a set of six primers developed previously by Zhang et al. [@doi:10.1101/2020.02.26.20028373].
-To determine assay sensitivity, serial tenfold dilutions of _in vitro_ transcribed _N_-gene RNA standard (IVT RNA) were tested using quantities from 10^5 copies down to 10 copies.
+Various combinations of reagents are available, but one example is the WarmStart Colorimetric LAMP 2X Master Mix with a set of six primers developed previously by Zhang et al. [@doi:10.1101/2020.02.26.20028373].
+To determine assay sensitivity, serial tenfold dilutions of _in vitro_ transcribed N-gene RNA standard were tested using quantities from 10^5^ copies down to 10 copies.
```

Comment:
This is the only instance where we don't italicize N. I think it should be it since it refers to the gene sequence here, not the protein.

Reply:
Fixed in c1298eea0dcc728014917c3ca9428558eba98072